tools/anthosbm-ansible-module/roles/anthos/tasks/main.yml

```diff
@@ -14,49 +14,77 @@
 
 - name: Reset ssh connection to allow group changes
   meta: reset_connection
-  
-- name: Delete bmctl workspace cluster directory if exists
-  ansible.builtin.file:
-    path: "{{ login_user_home }}/{{ bmctl_workspace_dir }}/{{ cluster_name }}"
-    state: absent
-    
+  tags: [preflight-admin, preflight-hybrid,create-admin,create-hybrid,create-user]
+      
 - name: Create bmctl workspace cluster directory
   ansible.builtin.file:
     path: "{{ login_user_home }}/{{ bmctl_workspace_dir }}/{{ cluster_name }}"
     state: directory
     owner: "{{ login_user }}"
     group: "{{ login_user_group }}"
+  tags: [preflight-admin,preflight-hybrid,preflight-user,create-admin,create-hybrid,create-user]  
 
 - name: Create Cluster Config file
   ansible.builtin.template:
-    src: cluster.yaml.j2
+    src: ../../shared-files/cluster_config.yaml.j2
```

Comment:
It would be better to keep it in the default subfolder (templates) of Role folder structure to avoid giving relative paths.

Reply:
Would this cause any issues if multiple playbooks referred to the same jinja template? I believe that was the reason to abstract it to a higher directory.

Reply:
It would not. The variables are resolved during runtime.

Reply:
@ravi-dalal we have more module that uses this template.. Like upgrade/update .. Thats why we thought of moving this in a common folder, so other playbooks can also call them. what do you think ?

Reply:
It would be good to keep copy in their template folder. I have seen that the absolute or relative paths causes issues.